.github/workflows/file_health.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+
+# Assumption: file names don't contain `:` (for the `cut` invocation).
+# Safe to assume, as such a character in a filename would cause trouble on Windows, a platform we support
+
+# any regression turn this non-zero
+regressions=0
+
+# exclusions (these are just grep extended regular expressions to match against paths relative to the root of the repository)
+exclusions_nonutf8='.*(7z|gif|ic(ns|o)|png|qm|zip)'
+exclusions_bom='src/base/unicodestrings.h'
+exclusions_tw='src/webui/www/private/scripts/lib/mootools-1.2-more.js'
+exclusions_no_lf='(compile_commands.json)|(src/webui/www/private/scripts/lib/mootools-1.2-(core-yc.js|more.js))|(.*svg)'
+
+echo -e "*** Detect files not encoded in UTF-8 ***\n"
+
+find . -path ./build -prune -false -o -path ./.git -prune -false -o -type f -exec file --mime {} \; | grep -E -v -e "charset=us-ascii" -e "charset=utf-8" -e "${exclusions_nonutf8}" | cut -d ":" -f 1 | sort | tee >(echo -e "\n--> Files not encoded in UTF-8: found" $(wc -l </dev/stdin) "regression(s)") | xargs -I my_input -0 bash -c 'echo "my_input"; test "$(echo -n "my_input" | wc -l)" -eq 0'; regressions=$((regressions+$?))
```

Comment:
Can you split them into mulitlines?
For example:
```bash
find . -path ./build -prune -false -o -path ./.git -prune -false -o -type f -exec file --mime {} \; \
    | grep -E -v -e "charset=us-ascii" -e "charset=utf-8" -e "${exclusions_nonutf8}" \
    | cut -d ":" -f 1 
```

Reply:
fixed.